# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/utf-16be.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/utf-16be.html.ini
--- a/testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/utf-16be.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/utf-16be.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -47,10 +47,9 @@
 
 [utf-16be.html?include=workers]
   expected:
-    if (os == "linux") and webrender and not swgl and not debug and not fission: ["TIMEOUT", "OK"]
-    if (os == "linux") and webrender and not swgl and debug: ["TIMEOUT", "OK"]
-    if (os == "linux") and webrender and swgl: ["TIMEOUT", "OK"]
-    if os == "mac": ["TIMEOUT", "OK"]
+    if (os == "linux") and not fission and (processor == "x86_64") and not swgl and not webrender and not debug: ["TIMEOUT", "OK"]
+    if (os == "linux") and not fission and (processor == "x86_64") and not swgl and webrender: ["TIMEOUT", "OK"]
+    if (os == "android") and debug and webrender: ["TIMEOUT", "OK"]
     [OK, TIMEOUT]
   [SharedWorker() in a shared worker]
     expected: FAIL
@@ -62,10 +61,9 @@
 
   [SharedWorker constructor]
     expected:
-      if (os == "linux") and webrender and not swgl and not debug and not fission: ["TIMEOUT", "PASS"]
-      if (os == "linux") and webrender and not swgl and debug: ["TIMEOUT", "PASS"]
-      if (os == "linux") and webrender and swgl: ["TIMEOUT", "PASS"]
-      if os == "mac": ["TIMEOUT", "PASS"]
+      if (os == "linux") and not fission and (processor == "x86_64") and not swgl and not webrender and not debug: ["TIMEOUT", "PASS"]
+      if (os == "linux") and not fission and (processor == "x86_64") and not swgl and webrender: ["TIMEOUT", "PASS"]
+      if (os == "android") and debug and webrender: ["TIMEOUT", "PASS"]
       [PASS, TIMEOUT]
 
   [SharedWorker() in a dedicated worker]
@@ -73,8 +71,8 @@
 
   [Worker() in a shared worker]
     expected:
+      if os == "win": ["PASS", "TIMEOUT"]
       if os == "mac": ["PASS", "TIMEOUT"]
-      if os == "win": ["PASS", "TIMEOUT"]
 
 
 [utf-16be.html?include=nested-browsing]